_data/compatibility/bitmex.yaml

```diff
@@ -34,11 +34,11 @@ segwit:
   features:
     receive:
       p2sh_wrapped: "false"
-      bech32: "false"
-      default: "p2sh"
+      bech32: "true"
+      default: "bech32"
     send:
       bech32: "true"   # https://blog.bitmex.com/bitmex-enables-bech32-sending-support/
-      change_bech32: "false"
+      change_bech32: "true"
       segwit_v1: "Bech32 not supported."
```

Comment:
According to this, BitMEX should also support bech32m now: https://twitter.com/BitMEXResearch/status/1492152557044654082

@bitschmidty: What's the story behind the `segwit_v1` fields? We don't show those yet, do we?

Reply:
ah yeah that's true as well